scripts/cc-cli/src/commands/wizard.ts

```diff
@@ -82,22 +85,22 @@ export function makeWizardCommand() {
     // Check both accounts have funds
     const stashBalance = await getBalance(stashAddress, api);
     const controllerBalance = await getBalance(controllerAddress, api);
-    checkStashBalance(stashAddress, stashBalance, options.amount);
-    checkControllerBalance(controllerAddress, controllerBalance, 1);
-    const bondExtra: boolean = checkIfAlreadyBonded(stashAddress, stashBalance);
+    checkStashBalance(stashAddress, stashBalance, amount);
+    checkControllerBalance(controllerAddress, controllerBalance, new BN(2));
```

Comment:
Q: Is 2 CTC always going to be enough to pay txn fees when bonding ?

Reply:
Yeah, if just bonding, it should def be enough. If they start interacting frequently and then need to unbond, for example, they might need more to avoid going below 1 CTC.